target/scripts/startup/setup-stack.sh

```diff
@@ -342,31 +343,98 @@ function _setup_dovecot_local_user
       if [[ -f /tmp/docker-mailserver/dovecot-quotas.cf ]]
       then
         declare -a USER_QUOTA
-        IFS=':' ; read -r -a USER_QUOTA < <(grep "${USER}@${DOMAIN}:" -i /tmp/docker-mailserver/dovecot-quotas.cf)
+        IFS=':' read -r -a USER_QUOTA < <(grep "${USER}@${DOMAIN}:" -i /tmp/docker-mailserver/dovecot-quotas.cf)
         unset IFS
```

Comment:
IFS always have a default values.
We don't change this value generally, just for the specific line 346.
If we unset IFS, then IFS is empty.
For example, this would be equally to putting "unset IFS" at the start of all scripts - pointless.

---

Before this change, there was a `;` after `IFS=':'`. This way, the default value was changed to `:`. Even then, restoring the old IFS value would be better instead of completely unseting it.

```suggestion
```

Reply:
`unset IFS` seems fine for bash to reset to default value, it will not make IFS empty:

- https://unix.stackexchange.com/questions/97481/ifs-null-is-not-the-same-as-unset-ifs/97492#97492
- https://stackoverflow.com/questions/62470380/why-cant-reset-ifs-variable/62470888#62470888

Reply:
> We also have unset IFS present in check-for-changes.sh, helper-functions.sh and setup-stack.sh already.

Removing the `;` before `read` like here would also allow to remove `unset IFS` there.

Reply:
@casperklein is right, removing the `unset IFS` here is a good addition. I'd just like to point out that this PR is ultimately not about removing `unset`s.